Assets/Scripts/Game.cs

```diff
@@ -72,6 +75,13 @@ void Start()
         // Create some things
         uiScaler = new UIScaler(uICanvas);
         config = new ConfigFile();
+        currentLang = 1; // English
+
+        if (config.data.Get("MoMConfig") != null)
```

Comment:
Better to use:

if (config.data.Get("MoMConfig", "currentLang") != "")

This allows for the config to exist but language not set (as would be for existing users).